src/UserLabel/UserLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import { IconWithLabel } from "../IconWithLabel";
+import React from "react";
+import { UserAvatar } from "../UserAvatar";
+import { UserLabelProps } from "./UserLabel.types";
+
+/**
+ * Component that renders a user avatar and the name of the user to the right
+ * @param label The name of the user to be displayed
+ */
+export default function UserLabel({ label }: UserLabelProps) {
+  return IconWithLabel({
+    icon: (
+      <div>
+        <UserAvatar
+          sx={{ height: "24px", width: "24px" }}
+          color="#EC407A"
```

Comment:
@dmbarry86 and @Sowbhagya-ipg I followed the current implementation of the road preview, where colour is hard coded, is this fine for now?

Reply:
We should avoid hard-coding any colours inside components. Instead, we should reference them centrally using the theme. You can extend the theme if it does not currently contain a prop for what you need.

Reply:
I have exposed the prop now that the color can be controlled. If no color prop is passed, the `UserAvatar` has a default value. I have updated storybook and added tests.